go/cmd/dolt/commands/log.go

```diff
@@ -399,34 +424,59 @@ func logCommits(ctx context.Context, dEnv *env.DoltEnv, opts *logOpts) int {
 	if err != nil {
 		cli.PrintErrln(err)
 		return 1
+	}*/
+	cHashToRefs, err := getHashToRefs(ctx, dEnv, opts.decoration, queryist, sqlCtx)
+	if err != nil {
+		return handleErrAndExit(err)
 	}
 
 	var commitsInfo []logNode
-	for _, comm := range commits {
-		meta, mErr := comm.GetCommitMeta(ctx)
+	for _, row := range sqlResult {
+		meta := &datas.CommitMeta{
+			Name:          row[1].(string),
+			Email:         row[2].(string),
+			Timestamp:     uint64(row[3].(time.Time).Unix()),
+			Description:   row[4].(string),
+			UserTimestamp: 0,
+		}
+		/*meta, mErr := comm.GetCommitMeta(ctx)
 		if mErr != nil {
 			cli.PrintErrln("error: failed to get commit metadata")
 			return 1
+		}*/
+
+		parents, err := GetRowsForSql(queryist, sqlCtx, fmt.Sprintf("select parent_hash from dolt_commit_ancestors where commit_hash = %s", row[0]))
```

Comment:
ready row[0] into a variable above. Generally when you have an array at the top of the block and you have static indexes for for particular values, you want to copy them all into named values right away. When I read the lines 435 - 438, my first thought was 'where is row[0]`